### PR TITLE
CI: for gh actions, use the uyuni containerized server

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
## What does this PR change?
We were building the server container since there had not yet been released the containerized server in uyuni. Now that is released, it is time to use that one.
## Links
Fixes https://github.com/SUSE/spacewalk/issues/22061
- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: testsuite/podman_runner/08_manager_setup.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@ set -xe
 
 src_dir=$(cd $(dirname "$0")/../.. && pwd -P)
 
-sudo -i podman exec uyuni-server-all-in-one-test bash -c "/usr/lib/susemanager/bin/mgr-setup -l /var/log/susemanager_setup.log -s"
+sudo -i podman exec -e CERT_O="test_org" -e CERT_OU="test_ou" -e CERT_CITY="test_city" -e CERT_STATE="test_state" -e CERT_COUNTRY="DE" -e CERT_EMAIL="a@b.com" -e CERT_CNAMES="uyuni-server-all-in-one-test" -e CERT_PASS="spacewalk" -e UYUNI_FQDN="uyuni-server-all-in-one-test" -e MANAGER_USER="admin" -e MANAGER_PASS="spacewalk" -e MANAGER_ADMIN_EMAIL="a@b.com" -e MANAGER_MAIL_FROM="a@b.com" -e MANAGER_ENABLE_TFTP="n" -e MANAGER_DB_NAME="manager" -e MANAGER_DB_HOST="localhost" -e MANAGER_DB_PORT="5432" -e MANAGER_DB_USER="manager" -e MANAGER_DB_PASS="manager" -e MANAGER_DB_PROTOCOL="TCP" -e REPORT_DB_NAME="reportdb" -e REPORT_DB_USER="report" -e REPORT_DB_PASS="report" -e EXTERNALDB_ADMIN_USER="" -e EXTERNALDB_ADMIN_PASS="" -e EXTERNALDB_PROVIDER="" -e ISS_PARENT="" -e ACTIVATE_SLP="" -e SCC_USER="" -e SCC_PASS="" uyuni-server-all-in-one-test bash -c "/usr/lib/susemanager/bin/mgr-setup -l /var/log/susemanager_setup.log -s"
```
[the remark this answers is not in the document]
Comment (jordimassaguerpla) [reply testsuite/podman_runner/08_manager_setup.sh:6]: Sure, I can give it a try.

[the remark this answers is not in the document]
Comment (jordimassaguerpla) [reply testsuite/podman_runner/08_manager_setup.sh:6]: where is the setup_env.sh script?

[the remark this answers is not in the document]
Comment (srbarrios) [reply testsuite/podman_runner/08_manager_setup.sh:6]: You remove it on this PR :)

File: testsuite/dockerfiles/server-all-in-one/Dockerfile [hunk 2]
```diff
@@ -1,35 +0,0 @@
...
-RUN zypper --non-interactive install --auto-agree-with-licenses --force-resolution patterns-uyuni_server
-RUN zypper mr -d repo-backports-update repo-sle-update
-
-# hack to correct the report db script to work on containers
-RUN cp /usr/bin/uyuni-setup-reportdb /usr/bin/uyuni-setup-reportdb.original
-RUN sed -i 's/sysctl kernel.shmmax/#sysctl kernel.shmmax/g' /usr/bin/uyuni-setup-reportdb
-RUN sed -i 's/-a "$PG_SOCKET"//g' /usr/bin/uyuni-setup-reportdb
-
-# hack to fix pam for podman
-COPY etc_pam.d_sshd /etc/pam.d/sshd
```
[the remark this answers is not in the document]
Comment (jordimassaguerpla) [reply testsuite/dockerfiles/server-all-in-one/Dockerfile:24]: Not really. It passed all the core and secondary tests, so we do not need it.

**Review by srbarrios — APPROVED** [review]

**Pull request closed without merging**